Source/NSubstitute/Core/CustomHandlers.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace NSubstitute.Core
+{
+    public class CustomHandlers : ICustomHandlers
+    {
+        private readonly ISubstituteState _substituteState;
+        private readonly List<ICallHandler> _handlers = new List<ICallHandler>();
+
+        public IEnumerable<ICallHandler> Handlers => _handlers;
+        public IDictionary<object, object> HandlerDataStorage { get; } = new ConcurrentDictionary<object, object>();
```

Comment:
No, it's not a strong requirement, it's just my preference to stick to one style. I think I just missed the point when we all switched to C# 6 😊 I call @dtchepak here